sycl/include/sycl/ext/oneapi/matrix/matrix-jit.hpp

```diff
@@ -248,6 +258,75 @@ class wi_element {
                         PI_INVALID_DEVICE);
 #endif // __SYCL_DEVICE_ONLY__
   }
+  wi_element &
```

Comment:
For the sake of visual distinction I would prefer a space between each operator overload.

Reply:
Sure, I've added them